release_notes/v2.0.0.txt

```diff
@@ -50,4 +50,4 @@ none
 
 Change Log
 ----------
-https://github.com/hyperledger/fabric-chaincode-node/blob/release-2.0/CHANGELOG.md#v2.0.0
+https://github.com/hyperledger/fabric-chaincode-node/blob/release-2.x/CHANGELOG.md#v2.0.0
```

Comment:
Should this be...

`https://github.com/hyperledger/fabric-chaincode-node/blob/release-2.x/CHANGELOG.md#v2.1.0`

?

Reply:
Not for the 2.0.0.txt release note :)

Reply:
Good point!